extensions/amp-carousel/0.1/slides.js

```diff
@@ -246,7 +246,7 @@ export class AmpSlides extends BaseCarousel {
     this.updateInViewport(oldSlide, false);
     this.updateInViewport(newSlide, true);
     this.setControlsState();
-    this.schedulePause(oldSlide);
+    this.scheduleUnload(oldSlide);
```

Comment:
So, rediscussed it one more time with Erwin, let's make it `schedulePause`. And the same in the carousel.js (we will actually remove it completely from there soon).